util/tooltool/tooltool.py

```diff
@@ -833,39 +836,39 @@ def upload(manifest, message, base_urls, auth_file, region):
     # Upload the files, each in a thread.  This allows us to start all of the
     # uploads before any of the URLs expire.
     threads = {}
-    for filename, file in files.iteritems():
-        if 'put_url' in file:
-            log.info("%s: starting upload", filename)
+    for filename, file_ in files.iteritems():
+        if 'put_url' in file_:
+            log.info("%s: starting upload", (filename,))
```

Comment:
What's the purpose of making this a tuple? I think you're working around the `% tuple` problem which doesn't apply to the logging module.

    >>> logging.info("%s", 123)
    123
    >>> logging.info("%s", (123,))
    (123,)
    >>> logging.info("%s", ((123,),))
    ((123,),)